environments/icds-cas/public.yml

```diff
@@ -286,15 +286,11 @@ localsettings:
     'static-icds-cas-static-mpr_3i_person_cases': 'static-icds-cas-static-mpr_3i_person_cases-optimized'
     'static-icds-cas-static-mpr_2a_deaths': 'static-icds-cas-static-mpr_2a_deaths-optimized'
     'static-icds-cas-static-mpr_2a_person_cases': 'static-icds-cas-static-mpr_2a_person_cases-optimized'
-    'static-icds-cas-static-mpr_2bi_preg_delivery_death_list': 'static-icds-cas-static-mpr_2bi_preg_delivery_death_list-optimized'
```

Comment:
@czue FYI going to remove this to reduce the queries/emails coming in. Can be added back later